src/components/CourseSection/CourseSection.jsx

```diff
@@ -53,6 +53,7 @@ class CourseSection extends React.Component {
                 key: courseData.course_run_id,
                 title: courseData.display_name,
                 microMastersTitle: courseData.micromasters_title,
+                hasEmailsEnabled: courseData.emails_enabled,
```

Comment:
Related to the bug I noticed (see above comment), we also need to be passing the `emails_enabled` field to all course card types.